apps/zui/src/js/state/Lakes/types.ts

```diff
@@ -6,6 +6,9 @@ export type LakeAttrs = {
   version?: string
   authType: AuthType
   authData?: AuthData
+  features?: {
+    describe: boolean
+  }
```

Comment:
I'm going to add a "features" property the the data we cache about the lake. The only feature right now will be "describe".